### DISCUSSION TITLE
StudioElf.RocketFuel 1.5.3 Released.

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#2664] [Show and tell]
This is a pre release just before the new Oqtane release, but still worth the update.  Update through the normal channel Oqtane Marketplace.

Add modules from the page.
![RF-Add-Module-Update](https://user-images.githubusercontent.com/9447612/224485503-2c6be667-3268-40aa-926d-d0aeda473a6d.gif)

Add modules out of edit mode.
![RF-Add-Module-Update-Capsual](https://user-images.githubusercontent.com/9447612/224485546-533716d5-38a5-4b29-bd4c-8dfc0b4ec0ea.gif)

New Naviagtor menu and formatted Page and Module information.
![RF-Add-Module-Update-Nav-Json](https://user-images.githubusercontent.com/9447612/224485589-d5588f12-c439-4d8c-8e84-27ffe1bb56ee.gif)

**Comment by larsontim12** [comment]
When I install RocketFuel on a new install of Oqtane through the Theme extensions and restart the application it comes up with all kinds of errors and I have to delete and go back to the base install to get it to work again.

**Comment by leigh-pointer** [comment]
@larsontim12 thanks for the report.
Released 3,4, Oqtane 3.4 compatable
